app/core/settings.py

```diff
@@ -88,6 +91,47 @@
     'DATETIME_FORMAT': "%Y-%m-%dT%H:%M:%SZ",
 }
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
```

Comment:
a JSON format similar to https://github.com/elifesciences/profiles/blob/develop/profiles/logging.py#L36 will be easier to filter and access on Loggly